docs/intentReaderHandlers/RunTestHandler.md

```diff
@@ -0,0 +1,338 @@
+# RunTestHandler 
+
+## Overview
+
+RunTestHandler is invoked when the task specified in the intent has the value "runTest". This handler is used to run sanity tests in FCA. When this handler is invoked, it will loop through all the APIs in the openRpc schema, invoke the APIs, and perform schema validations on the responses. Once the execution is over, a test report is generated. For more about execution, [click here](../Execution.md#sanity-test-execution) and for reporting [here.](../Reporting.md#supported-ways-of-retrieving-reports) 
+
+It performs the following actions:
```

Comment:
In my window this is not formatting the numbered list correctly:
It performs the following actions: 1. Parses the input message received to fetch the action and parameters. 2. Loops through the APIs in openRpc.  3. Saves the API response or error and performs schema validation. 4. Formats the result and sends the response back to IntentReader.